aea/cli/add.py

```diff
@@ -148,6 +211,12 @@ def add_item(ctx: Context, item_type: str, item_public_id: PublicId) -> None:
 
     _add_item_deps(ctx, item_type, item_config)
     register_item(ctx, item_type, item_config.public_id)
+
+    ipfs = ctx.agent_config.ipfs
+    ipfs[f"{item_type}s"][str(item_config.public_id)] = {
+        "hash": item_config.package_hash
+    }
```

Comment:
We cannot add the hash inside the package; that changes the package hash. Instead, we should keep a local list of public ids to hashes; that can be saved in a local file for now